site/cypress/e2e/grading_stats.spec.js

```diff
@@ -0,0 +1,111 @@
+import { skipOn } from '@cypress/skip-test';
+
+skipOn(Cypress.env('run_area') === 'CI', () => {
```

Comment:
Is there a reason why this can't run in the CI?  Tests which aren't run regularly as part of our CI are almost useless because they quickly become outdated.  Many of the Selenium tests aren't run as part of the CI for historical reasons, but our CI is now more complete and should be able to support almost all tests.  If we can't support the entire test, can some subset of the test be enabled at least?